sdk/scheduler/src/main/java/com/mesosphere/sdk/framework/OfferProcessor.java

```diff
@@ -396,15 +394,10 @@ private void destroyFramework() {
         }
 
         LOGGER.info("Tearing down framework...");
-        Optional<SchedulerDriver> driver = Driver.getDriver();
-        if (driver.isPresent()) {
-            // Stop the SchedulerDriver thread:
-            // - failover==false: Tells Mesos to teardown the framework.
-            // - This call will cause FrameworkRunner's SchedulerDriver.run() call to return DRIVER_STOPPED.
-            driver.get().stop(false);
-        } else {
-            LOGGER.error("No driver is present for deregistering the framework.");
-        }
+        // Stop the SchedulerDriver thread:[
```

Comment:
(nit) remove trailing `[` Ignore this comment if you don't end up making more commits to this PR.

Reply:
Will fix in a following PR

Reply:
( Fixing via #2592 )